paper-autocomplete.html

```diff
@@ -206,6 +206,43 @@
 </dom-module>
 <script>
   (function () {
+    //---------------------------------------------------------------------------
+    //                  IE11 function polyfills to work properly
+    //---------------------------------------------------------------------------
+
+    if (window.NodeList && !NodeList.prototype.forEach) {
+      NodeList.prototype.forEach = function (callback, argument) {
+          argument = argument || window;
+          for (var i = 0; i < this.length; i++) {
+              callback.call(argument, this[i], i, this);
+          }
+      };
+    }
+
+    if (!Array.prototype.find) {
```

Comment:
I think we should not add this polyfill. `Array.prototype.find` does not add such an important feature since you can basically achieve the same using `Array.prototype.filter` and then selecting the first element